src/ai/susi/server/api/aaa/ChangeUserRoles.java

```diff
@@ -0,0 +1,112 @@
+ *  1. user    --> The username of the user whose role is to be modified
+ *  2. role    --> the new role of the user
+ */
+
+public class ChangeUserRoles extends AbstractAPIHandler implements APIHandler {
+    private static final long serialVersionUID = -1432553481906185711L;
+
+    @Override
+    public BaseUserRole getMinimalBaseUserRole() { return BaseUserRole.ADMIN; }
+
+    @Override
+    public JSONObject getDefaultPermissions(BaseUserRole baseUserRole) {
+        return null;
+    }
+
+    @Override
+    public String getAPIPath() {
+        return "/aaa/changeRoles";
+    }
+
+
+    @Override
+    public JSONObject serviceImpl(Query query, HttpServletResponse response, Authorization authorization, final JsonObjectWithDefault permissions) throws APIException {
+
+        JSONObject result = new JSONObject();
+        if (authorization.getBaseUserRole().toString().equals("ADMIN")) {
+            if (query.get("getServicePermissions", null) != null) {
+                String serviceString = query.get("getServicePermissions", null);
+
+                Class<?> serviceClass;
+                try {
+                    serviceClass = Class.forName(serviceString);
+                } catch (ClassNotFoundException e) {
+                    throw new APIException(400, "Bad service name (no class)");
+                }
+                Constructor<?> constructor;
+                try {
+                    constructor = serviceClass.getConstructors()[0];
+                } catch (Throwable e) {
+                    throw new APIException(400, "Bad service name (no constructor)");
+                }
+                Object service;
+                try {
+                    service = constructor.newInstance();
+                } catch (Throwable e) {
+                    throw new APIException(400, "Bad service name (no instance possible)");
+                }
+
+                if (service instanceof AbstractAPIHandler) {
+                    result.put("servicePermissions", authorization.getPermissions((AbstractAPIHandler) service));
+                    return result;
+                } else {
+                    throw new APIException(400, "Bad service name (no instance of AbstractAPIHandler)");
+                }
+            } else if (query.get("getServiceList", false)) {
+                JSONArray serviceList = new JSONArray();
+                for (Class<? extends Servlet> service : SusiServer.services) {
+                    serviceList.put(service.getCanonicalName());
+                }
+                result.put("serviceList", serviceList);
+                return result;
+            } else if (query.get("getUserRolePermission", false)) {
+                result.put("userRolePermissions", authorization.getUserRole().getPermissionOverrides());
+                return result;
+            } else {
+                // Accept POST params
+                String userTobeUpgraded = query.get("user", null);
```

Comment:
Yes. Its possible now but why will he do so. 
Also the first Admin has to make himself the admin by changing the value of userRole in the db/authorization.json.
For example, first orbiter can be the admin by changing the value of his userRole in authorization.json and then he can give rights to other admins.